lib/perl/Genome/Model/ClinSeq/Command/CreateMutationSpectrum-exome.t

```diff
@@ -14,27 +14,35 @@ BEGIN {
 
 use above "Genome";
 use Test::More tests => 11;  #One per 'ok', 'is', etc. statement below
-use Genome::Model::ClinSeq::Command::CreateMutationSpectrum;
 use Data::Dumper;
+use Genome::Model::ClinSeq::TestData;
+use Genome::Utility::Test;
+use Sub::Override;
 
-use_ok('Genome::Model::ClinSeq::Command::CreateMutationSpectrum') or die;
+my $pkg = 'Genome::Model::ClinSeq::Command::CreateMutationSpectrum';
+use_ok($pkg);
+
+use Genome::Model::Build::ReferenceSequence;
+my $override = Sub::Override->new(
+    'Genome::Model::Build::ReferenceSequence::full_consensus_path',
+    sub { return '/gscmnt/ams1102/info/model_data/2869585698/build106942997/all_sequences.fa'; }
```

Comment:
I was hoping to discuss this with you. The tool needs a reference sequence in order to run. This is the path to the reference sequence that the original test build uses. I don't want to put it in the clinseq test data because it's such a large file so I wasn't sure how to proceed so I hardcoded the path for now.